docs/en/integrations/data-ingestion/kafka/index.md

```diff
@@ -823,6 +868,7 @@ ClickHouse Kafka Connect reports the following metrics:
 - Batch size is inherited from the Kafka Consumer properties.
 - When using KeeperMap for exactly-once and the offset is changed or rewound, you need to delete the content from KeeperMap for that specific topic.
```

Comment:
Can you please provide how to do it or a link to somewhere in docs that this explained

Reply:
Is this resolved yet?